modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/GlassViewEventHandler.java

```diff
@@ -60,22 +60,17 @@
 import javafx.scene.input.TransferMode;
 import javafx.scene.input.ZoomEvent;
 
-import java.security.AccessController;
-import java.security.PrivilegedAction;
+import java.util.function.Supplier;
 
 class GlassViewEventHandler extends View.EventHandler {
 
     static boolean zoomGestureEnabled;
     static boolean rotateGestureEnabled;
     static boolean scrollGestureEnabled;
```

Comment:
```java
    private static final boolean zoomGestureEnabled = Boolean.valueOf(System.getProperty("com.sun.javafx.gestures.zoom", "false"));
    private static final boolean rotateGestureEnabled = Boolean.valueOf(System.getProperty("com.sun.javafx.gestures.rotate", "false"));
    private static final boolean scrollGestureEnabled = Boolean.valueOf(System.getProperty("com.sun.javafx.gestures.scroll", "false"));
```

Reply:
Yes, I could have done that, but chose to minimize the diffs in this case. Now that it is integrated, it's probably not worth revisiting.